.github/workflows/lambda-release-tag-runners.yml

```diff
@@ -1,3 +1,6 @@
+on:
+  workflow_dispatch: {}
```

Comment:
I guess, you don't need `{}`.

Reply:
yeah, some places have it, others don't; As there is no pattern, I'll keep like this (because I find it more readable) :)